examples/test_data/download_utils.py

```diff
@@ -31,12 +31,27 @@
 import hashlib
 import io
 import time
+import os
 
 # Typically "Open3D/examples/test_data", the test data dir.
 _test_data_dir = Path(__file__).parent.absolute().resolve()
 
 # Typically "Open3D/examples/test_data/open3d_downloads", the download dir.
-_download_dir = _test_data_dir / "open3d_downloads"
+_download_dir = os.path.join(_test_data_dir, "open3d_downloads")
+
+
+def get_data_path_common(relative_path=""):
```

Comment:
Should we just return the common path here, and add relative path outside?

Reply:
I think, this interface might be more convenient for users, also it is consistent with C++ version (DataManager.h).